composer.json

```
@@ -34,7 +34,7 @@
         }
     ],
     "require": {
-        "flarum/core": "^1.2.0"
+        "flarum/core": "^1.8.7"
```

```suggestion
        "flarum/core": "^1.8.6"
```
Although 1.8.6 is buggy with the basicspage, technically this is the minimum supported version for these changes

---

Yeah I agree, but the CI won't pass if we use 1.8.6, see https://github.com/FriendsOfFlarum/impersonate/actions/runs/11156322074/job/31450506821

---

How random! Ok, no worries then :)